tests/ja4-rules-requires-off/test.yaml

```diff
@@ -1,10 +1,11 @@
 requires:
-  min-version: 8.0.0
+  min-version: 7.0.6
   script:
     - ./src/suricata --build-info | grep "JA4 support" | grep no > /dev/null
 
 args:
   - -k none
+  - --set logging.default-log-level=info
```

Comment:
Why this addition ?

Reply:
Is it Info in 7, and something else in 8 ?

Reply:
It was missing from the original test; the same level is used for 7 and 8.

Reply:
So why do we need it ?
Tests were passing for 8...